packages/wonder-blocks-modal/components/modal-launcher.md

```diff
@@ -152,3 +151,77 @@ const oneColumnModal = ({closeModal}) => <OneColumnModal
     </ModalLauncher>
 </View>;
 ```
+
+## Triggering programmatically
+
+Sometimes you'll want to trigger a modal programmatically.  This can be done
+by rendering `ModalLauncher` without any children and instead setting its
+`opened` prop to `true`.  In this situation `ModalLauncher` is a controlled
+component which means you'll also have to update `opened` to `false` in
+response to the `onClose` callback being triggered.
+
+```js
+const React = require("react");
+
+const {Title} = require("@khanacademy/wonder-blocks-typography");
+const {View} = require("@khanacademy/wonder-blocks-core");
+const Button = require("@khanacademy/wonder-blocks-button").default;
+const {ActionMenu, ActionItem} = require("@khanacademy/wonder-blocks-dropdown");
+
+class Example extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            opened: false,
+        };
+    }
+
+    handleOpen() {
+        console.log('opening modal');
+        this.setState({opened: true});
+    }
+
+    handleClose() {
+        console.log('closing modal');
+        this.setState({opened: false});
+    }
+
+    render() {
+        return <View>
+            <ActionMenu menuText="actions">
+                <ActionItem
+                    label="Open modal"
+                    onClick={() => this.handleOpen()}
+                />
+            </ActionMenu>
+            <ModalLauncher
+                key="clearFlags"
```

Comment:
Do we need this ref?

Reply:
Nope.  Must've been copy/pasta from somewhere.  I'll remove this.